azure-stack/hci/concepts/system-requirements.md

```diff
@@ -72,6 +72,9 @@ Here are the Azure requirements for your Azure Stack HCI cluster:
 
 A standard Azure Stack HCI cluster requires a minimum of one server and a maximum of 16 servers.
 
+> [!IMPORTANT]
+> When designing the number of nodes in your cluster, make sure to consider how both [Windows Failover Cluster and Storage Spaces Direct quorum](quorum.md) will impact your cluster's availability.
+
 Keep the following in mind for various types of Azure Stack HCI deployments:
 
```

Comment:
```suggestion
- An Azure Stack HCI system requires a minimum of one server and a maximum of 16 servers. If you're using 2-4 servers, we recommend using a [witness](quorum.md) to help determine which server is most up to date if there is contention.
```

Reply:
Thanks @JasonGerend! 

The trigger for opening this PR was that a customer did not understand that Windows Failover Cluster and Storage Spaces Direct manage quorum separately. Whereas a Windows Failover Cluster allows you to configure an external witness resource for maintaining quorum, S2D does not. I worry that changing the wording as above will result in customers configuring WFC quorum but not considering how a node failure could impact S2D availability. I couldn't think of how to spell this out concisely and fell back on linking to the quorum article... 

Another way to address this could be a page dedicated to cluster sizing, multiple clusters, and related considerations-- which customers also seem to be asking for

Reply:
Good feedback on that table - I agree it's a problem and that we're not yet clear enough on pool quorum. I'm asking some of the PMs for more info and will keep you posted! :)

Reply:
@JasonGerend Did you have any luck getting more information from the PG?

Reply:
@jasongerend Have there been any updates on this conversation from the PG?